src/components/ExternalLink.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {Text} from 'react-native';
+import PropTypes from 'prop-types';
+import openURLInNewTab from '../libs/openURLInNewTab';
+
+const propTypes = {
+    /** External URL we want to link */
+    href: PropTypes.string.isRequired,
+
+    /** Any children to display */
+    children: PropTypes.node,
```

Comment:
I think that since you're putting the children inside a `Text` component, this component will only work if the children is a plain string. I also think it should be required, because without the children there will be nothing to render or click on.

```suggestion
    children: PropTypes.string.isRequired,
```